[2024-11-21T00:22:48.927+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-21T00:22:48.938+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_heart_disease_data.download_data manual__2024-11-21T00:22:47.549794+00:00 [queued]>
[2024-11-21T00:22:48.943+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_heart_disease_data.download_data manual__2024-11-21T00:22:47.549794+00:00 [queued]>
[2024-11-21T00:22:48.944+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-21T00:22:48.953+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): download_data> on 2024-11-21 00:22:47.549794+00:00
[2024-11-21T00:22:48.960+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4355) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-21T00:22:48.959+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'process_heart_disease_data', 'download_data', 'manual__2024-11-21T00:22:47.549794+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpag44xp2n']
[2024-11-21T00:22:48.961+0000] {standard_task_runner.py:72} INFO - Started process 4368 to run task
[2024-11-21T00:22:48.961+0000] {standard_task_runner.py:105} INFO - Job 70: Subtask download_data
[2024-11-21T00:22:48.992+0000] {task_command.py:467} INFO - Running <TaskInstance: process_heart_disease_data.download_data manual__2024-11-21T00:22:47.549794+00:00 [running]> on host 2276f79c0942
[2024-11-21T00:22:49.042+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_heart_disease_data' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-21T00:22:47.549794+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-21T00:22:47.549794+00:00'
[2024-11-21T00:22:49.043+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-21T00:22:49.081+0000] {logging_mixin.py:190} INFO - Plik zosta≈Ç pobrany i zapisany jako 'data.csv'
[2024-11-21T00:22:49.082+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-21T00:22:49.088+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-21T00:22:49.089+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=process_heart_disease_data, task_id=download_data, run_id=manual__2024-11-21T00:22:47.549794+00:00, execution_date=20241121T002247, start_date=20241121T002248, end_date=20241121T002249
[2024-11-21T00:22:49.135+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-21T00:22:49.152+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-21T00:22:49.154+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
